Marlin/Marlin_main.cpp

```diff
@@ -9496,39 +9496,6 @@ inline void gcode_M502() {
 
 #endif // ADVANCED_PAUSE_FEATURE
 
-#if ENABLED(MK2_MULTIPLEXER)
-
-  inline void select_multiplexed_stepper(const uint8_t e) {
-    stepper.synchronize();
-    disable_e_steppers();
-    WRITE(E_MUX0_PIN, TEST(e, 0) ? HIGH : LOW);
-    WRITE(E_MUX1_PIN, TEST(e, 1) ? HIGH : LOW);
-    WRITE(E_MUX2_PIN, TEST(e, 2) ? HIGH : LOW);
-    safe_delay(100);
-  }
-
-  /**
-   * M702: Unload all extruders
-   */
-  inline void gcode_M702() {
-    for (uint8_t s = 0; s < E_STEPPERS; s++) {
-      select_multiplexed_stepper(e);
-      // TODO: standard unload filament function
-      // MK2 firmware behavior:
-      //  - Make sure temperature is high enough
-      //  - Raise Z to at least 15 to make room
-      //  - Extrude 1cm of filament in 1 second
-      //  - Under 230C quickly purge ~12mm, over 230C purge ~10mm
-      //  - Change E max feedrate to 80, eject the filament from the tube. Sync.
-      //  - Restore E max feedrate to 50
-    }
-    // Go back to the last active extruder
-    select_multiplexed_stepper(active_extruder);
-    disable_e_steppers();
-  }
-
-#endif // MK2_MULTIPLEXER
```

Comment:
My hope is that this feature will be 100% compatible with the Prusa MK2, so that it can simply be installed on that machine and existing G-code will continue to work. There are other features of that firmware that also need to be implemented, but this piece of hardware is quite straightforward.

Until we've completed the implementation and found it to work with Prusa-oriented G-code, I feel it is too soon to move code into the stepper ISR. And even afterward I will continue to veto adding a multiplexer-oriented code to the stepper ISR. We're trying at all times to reduce the ISR, and obviously this is not vital, because `synchronize()` takes care of it.

You will also note that the multiplexer is not specified to switch rapidly, and that doing so may even be a good way to destroy the hardware.

My hope for the long term is that Prusa will no longer need to write their own divergent firmware. Apart from this feature there are still 4 or 5 other very useful features in their firmware that also need integration.

But… gotta get Marlin 2.0.x going first! That is our current priority.